go.mod

```diff
@@ -67,6 +67,7 @@ require (
 	github.com/uber/jaeger-lib v2.4.0+incompatible
 	github.com/vektra/mockery v0.0.0-20181123154057-e78b021dcbb5
 	github.com/wadey/gocovmerge v0.0.0-20160331181800-b5bfa59ec0ad
+	github.com/xdg/scram v0.0.0-20180814205039-7eeb5667e42c
```

Comment:
This redirects to `github.com/xdg-go/scram`

Reply:
Yes, actually, this line generate via `make fmt`,
and `github.com/xdg/scram` is dependent on `github.com/Shopify/sarama` (https://github.com/Shopify/sarama/blob/v1.22.1/go.mod#L14)

in jaeger `go.mod`, it has already existed that `github.com/Shopify/sarama`

Reply:
My point is that we should use xdg-go instead.

Reply:
The library's README recommends `import "github.com/xdg/scram"`. Since there are no sub-packages, it might work with both imports. I opened a ticket there https://github.com/xdg-go/scram/issues/3.

Reply:
Not sure GitHub allows recycling previously used names like npm does (or used to), but if so, this opens a relatively serious security window. So, if we trust code that is under `xdg-go`, we just shouldn't use `xdg`.